bin/aca_py.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+
+
+def find_python_executable():
+    if "PYTHON" in os.environ:
+        return os.environ["PYTHON"]
+    python_executable = subprocess.getoutput("command -v python3")
+    if not python_executable:
+        python_executable = "python"
+    return python_executable
+
+
+def main():
+    python_executable = find_python_executable()
+    aries_cloudagent_args = sys.argv[1:]
+    subprocess.run(
+        [python_executable, "-m", "aries_cloudagent"] + aries_cloudagent_args
+    )
```

Comment:
I don't know that this is actually a problem but just calling it out: this will result in two python processes running for the lifetime of the program. The original shell script would have been a shell process and a python process. Perhaps it would be better to `os.execv()`, resulting in only a single process?